### PR TITLE
anaconda initial release

### PR DESCRIPTION
**Pull request by M-Waszkiewicz-Anaconda**
dependency for asammdf

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -40,6 +40,8 @@ about:
   license: GPL-3.0
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: The `license` is `MPL-1.1 AND GPL-3.0-or-later AND LGPL-2.1-or-later`
see https://github.com/PyYoshi/cChardet/blob/master/COPYING

File: recipe/meta.yaml [hunk 2]
```diff
@@ -40,6 +40,8 @@ about:
...
   license_family: GPL
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: `license_family: Other`

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,27 +1,27 @@
...
 
 source:
-  url: https://pypi.io/packages/source/c/cchardet/cchardet-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: c428b6336545053c2589f6caf24ea32276c6664cb86db817e03a94c60afa0eaf
 
 build:
-  number: 4
-  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:13]: `skip: true  # [py<36]`, see https://github.com/PyYoshi/cChardet/blob/13076a3ab134c089b3460189a7d01d15b3f40f61/setup.py#L132

File: recipe/meta.yaml (hunk 1 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml]: Here is a confirmation https://github.com/PyYoshi/cChardet/blob/13076a3ab134c089b3460189a7d01d15b3f40f61/setup.py#L125-L127

**Review by skupr-anaconda — APPROVED** [review]

File: recipe/meta.yaml (hunk 3 above)
Comment (M-Waszkiewicz-Anaconda) [reply recipe/meta.yaml:13]: done

File: recipe/meta.yaml (hunk 1 above)
Comment (M-Waszkiewicz-Anaconda) [reply recipe/meta.yaml]: done

File: recipe/meta.yaml (hunk 2 above)
Comment (M-Waszkiewicz-Anaconda) [reply recipe/meta.yaml]: done

**Review by SC426 — APPROVED** [review]

**Review by ELundby45 — APPROVED** [review]

**Pull request merged**